policy.md

```diff
@@ -0,0 +1,19 @@
+# Policies and conventions
+
+## Traditionally hex settings should use JSON integers, not JSON strings
+
+The config JSON isn't enough of a UI to be worth jumping through string ↔ integer hoops to support an 0x… form ([source][integer-over-hex]).
```

Comment:
On Tue, Jan 05, 2016 at 07:56:12AM -0800, Vincent Batts wrote:
> > +The config JSON isn't enough of a UI to be worth jumping through string ↔ integer hoops to support an 0x… form ([source][integer-over-hex]).
> 
> perhaps this could be clearer as to what the expectation is.

Sure.  In c41523b → 408412b, I've called out network.classID as an
example (although it's currently using a string.  If we accept the
integer style that seems popular in [1], I'll file a PR updating
classID).

[1]: https://github.com/opencontainers/specs/pull/267#discussion_r48287582